pkg/compactor/syncer_metrics.go

```diff
@@ -1,124 +0,0 @@
-package compactor
-
-import (
-	"github.com/go-kit/log/level"
-	"github.com/prometheus/client_golang/prometheus"
-	"github.com/prometheus/client_golang/prometheus/promauto"
-
-	"github.com/cortexproject/cortex/pkg/util"
-	util_log "github.com/cortexproject/cortex/pkg/util/log"
-)
-
-// Copied from Thanos, pkg/compact/compact.go.
-// Here we aggregate metrics from all finished syncers.
-type syncerMetrics struct {
-	metaSync                  prometheus.Counter
-	metaSyncFailures          prometheus.Counter
-	metaSyncDuration          *util.HistogramDataCollector // was prometheus.Histogram before
-	metaSyncConsistencyDelay  prometheus.Gauge
-	garbageCollections        prometheus.Counter
-	garbageCollectionFailures prometheus.Counter
-	garbageCollectionDuration *util.HistogramDataCollector // was prometheus.Histogram before
-	compactions               prometheus.Counter
-	compactionRunsStarted     prometheus.Counter
-	compactionRunsCompleted   prometheus.Counter
-	compactionFailures        prometheus.Counter
-	verticalCompactions       prometheus.Counter
-}
-
-// Copied (and modified with Cortex prefix) from Thanos, pkg/compact/compact.go
-// We also ignore "group" label, since we only use a single group.
-func newSyncerMetrics(reg prometheus.Registerer) *syncerMetrics {
-	var m syncerMetrics
-
-	m.metaSync = promauto.With(reg).NewCounter(prometheus.CounterOpts{
-		Name: "cortex_compactor_meta_syncs_total",
-		Help: "Total blocks metadata synchronization attempts.",
-	})
-	m.metaSyncFailures = promauto.With(reg).NewCounter(prometheus.CounterOpts{
-		Name: "cortex_compactor_meta_sync_failures_total",
-		Help: "Total blocks metadata synchronization failures.",
-	})
-	m.metaSyncDuration = util.NewHistogramDataCollector(prometheus.NewDesc(
-		"cortex_compactor_meta_sync_duration_seconds",
-		"Duration of the blocks metadata synchronization in seconds.",
-		nil, nil))
-	m.metaSyncConsistencyDelay = promauto.With(reg).NewGauge(prometheus.GaugeOpts{
-		Name: "cortex_compactor_meta_sync_consistency_delay_seconds",
-		Help: "Configured consistency delay in seconds.",
-	})
```

Comment:
I couldn't find this metrics on the new compactor_metrics. Are we removing it?

Reply:
Good catch. Will add it to new location